packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -94,7 +94,7 @@ fetch = globalThis.fetch;
 const CustomRequest = class extends globalThis.Request {
   constructor(input, init) {
     if (init) {
-      init.cache = undefined;
+      delete init.cache;
```

Comment:
Using `Object.defineProperty()` would throw later on l88.

Reply:
I find this and l88 quite cryptic 🥲, hopefully this code will be improved/made clearer at some point